lib/trln_argon/bookmarks_controller_behavior.rb

```diff
@@ -49,6 +53,15 @@ def sms_path(options = {})
       def sms_url(options = {})
         sms_bookmarks_url(options)
       end
+
+      def bookmarks_share_url
+        bookmarked_ids = ''
+        current_bookmarks.each do |bookmark|
```

Comment:
Would something like this work? ```bookmarked_ids = current_bookmarks.join('|')``` then you can skip the each block and gsub to remove the traling pipe.